src/internal/modules/charge-information/controllers/charge-element.js

```diff
@@ -16,7 +16,7 @@ const actions = require('../lib/actions');
 const getBackLink = request => {
   const { step, licenceId, elementId } = request.params;
   const { chargeVersionWorkflowId } = request.query;
-  if (request.query.returnToCheckData === 1) {
+  if (request.query.returnToCheckData) {
```

Comment:
When I was testing this part it failed and appeared to be a bug because the query param returnToCheckData is defined as a boolean value at the route with a default value of false. That's the reason for the change.